src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,7 +114,16 @@ detailed extensive descriptions of these coordinate systems on the
 Restricted keywords for the `<CoordSysType>CoordinateSystem` field in the
 `coordsystem.json` file for EEG datasets:
 
--   `CapTrak`: RAS orientation and the origin between the ears
+-   `CapTrak`: RAS orientation and the origin approximately between LPA and RPA
```

Comment:
Speaking personally, I would rather BIDS introduce acronyms like LPA/RPA/LHJ/RHJ before using them.

Reply:
We're planning on adding a description of PA in the EEGLAB wiki page, but would that still be insufficient in your view? I think BIDS spec has a page discussing fiducials vs. anatomical landmarks, maybe we can reference that here?

Reply:
I added a section above linking to FAQ pages. Feedback welcome.